packages/components/jest.config.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console */
+/** @type {import('jest').Config} */
+
+import { JestConfigWithTsJest } from "ts-jest"
+import sharedConfig from "../../jest.config.js"
+
+const jestConfig: JestConfigWithTsJest = {
+  ...sharedConfig,
+  roots: ["<rootDir>"],
+  modulePathIgnorePatterns: [],
+  moduleNameMapper: {
+    ...sharedConfig.moduleNameMapper,
+    "@components/(.*)$": "<rootDir>/src/$1",
+    "@utils/(.*)$": "<rootDir>/utils/$1",
+  },
+}
+
+export default jestConfig
+
+process.env.TZ = "UTC"
+
+if (process.env.USE_REACT_16 === "true") {
```

Comment:
Not sure if we can centralise this, a task for later. Ideally we try get these removed though lol